src/Illuminate/Support/Collection.php

```diff
@@ -920,6 +920,23 @@ public function slice($offset, $length = null)
         return new static(array_slice($this->items, $offset, $length, true));
     }
 
+     /**
```

Comment:
StyleCI is correct according to Laravel's rules. The phpdoc must be aligned to match the method. There is no fixer to change the alignment of the actual method definition yet.